<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="text">
    <string>Image Label</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load_8bit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Load 8bit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load_12bit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>240</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Load 12bit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
